app/actions/transactionHistoryActions.js

```diff
@@ -14,7 +14,7 @@ type Props = {
 export const ID = 'TRANSACTION_HISTORY'
 
 export default createRequestActions(ID, ({ net, address }: Props = {}) => async (state: Object) => {
-  const transactions = await api.loadBalance(api.getTransactionHistoryFrom, { net, address })
+  const transactions = await api.neonDB.getTransactionHistory(net, address)
```

Comment:
Is this the only API that has issues with neoscan? I wasn't able to single out the point-of-failure in my tests, but I'm sure you guys were able to confirm that...

Reply:
Good question.  I'll verify after merging this change in.

Reply:
Cool, I'm testing now, as well...

Reply:
Looks solid to me! I haven't been able to repro the issue... 👍